player.cpp

```diff
@@ -0,0 +1,31 @@
+#include "player.h"
+
+Player::Player()
+{
+
+}
+
+void Player::intitialize(Ui::MainWindow *ui_ptr)
+{
+    ui = ui_ptr;
+}
+
+void Player::display_choice(Choices player_choice)
```

Comment:
I think this should be get_choice() function, the display thing should be handle in mainwindow (follow SOLID principle)

Reply:
I'll change the name, and I'll check out that SOLID principle.

Reply:
Moved display_round() and display_result() to mainwindow.